.chloggen/instana-cafile-config-option.yaml

```diff
@@ -0,0 +1,11 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
```

Comment:
There are no user-visible changes now, only new tests. Therefore, we don't need a changelog entry.